### PR TITLE
Teleblocks some areas to prevent abuse, hopefully this works, DNM until i test it a bit

### PR DESCRIPTION
**Pull request by F-OS**
:cl:
add: Teleblocks areas such as the nuke shuttle, the vault, and the armory
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) 
There is no reason for science to teleport into areas such as the nuke shuttle, the armory, or the vault.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
>KorPhaeron: can we not arbitrarily make random rooms teleporter proof to try and acccomodate a broken feature

**Comment by spectralexplorer** [comment]
main issue with teleblocking the nuke shuttle is that now instead of telesciencing the nuke away you do the same process to find the shuttle (throwing GPSes into space and hoping it hits) and then you just send someone out after it, more convoluted but still the same essential problem

**Comment by F-OS** [comment]
Sorry this was dumb and broken, closing it

**Comment by XDTM** [comment]
Then they have to face the guns at least

**Comment by PKPenguin321** [comment]
make a goofy machine that can be placed in the area that is teleblocked that makes it teleblocked

also freeze

File: code/modules/telesci/telesci_computer.dm [hunk 1]
```diff
@@ -287,6 +287,10 @@
 		telefail()
 		temp_msg = "ERROR! Sector is outside known time and space!"
 		return
+	if(A.teleporter_blocked == 1)
```
Comment (RemieRichards) [inline code/modules/telesci/telesci_computer.dm]: don't need the ```== 1```

File: code/game/area/areas/shuttles.dm [hunk 2]
```diff
@@ -51,10 +51,12 @@
 /area/shuttle/syndicate
 	name = "Syndicate Infiltrator"
 	blob_allowed = FALSE
+	teleporter_blocked = 1
```
Comment (RemieRichards) [inline code/game/area/areas/shuttles.dm]: use ```TRUE```, not ```1``` wherever you have.

File: code/game/area/areas.dm [hunk 3]
```diff
@@ -9,7 +9,7 @@
 	layer = AREA_LAYER
 	mouse_opacity = 0
 	invisibility = INVISIBILITY_LIGHTING
-
+	teleporter_blocked = 0
```
Comment (RemieRichards) [inline code/game/area/areas.dm]: use ```FALSE```, not ```0``` wherever you have.

**Comment by F-OS** [comment]
@RemieRichards Defucked the shitcode i made at 10pm.

**Comment by coiax** [comment]
I do not approve of arbitrary area restrictions with no IC explanation other than OOC balance.

**Comment by lzimann** [comment]
This is an awful way of dealing with telesci, arbitrary blocking an area without any justification in-game. Plus, this is a balance change and we are in a freeze.

**Comment by F-OS** [comment]
@lzimann alright, sorry, created this while tired, bad idea in retrospect

**Comment by XDTM** [comment]
>Fix telesci or we remove it
>No rebalancing telesci we're in a freeze

Can we also block balance-oriented removal PRs under the freeze

**Pull request closed without merging**